command/v7/create_service_command_test.go

```diff
@@ -0,0 +1,175 @@
+package v7_test
+
+import (
+	"code.cloudfoundry.org/cli/actor/actionerror"
+	"code.cloudfoundry.org/cli/actor/v7action"
+	"code.cloudfoundry.org/cli/command/commandfakes"
+	v7 "code.cloudfoundry.org/cli/command/v7"
+	"code.cloudfoundry.org/cli/command/v7/v7fakes"
+	"code.cloudfoundry.org/cli/resources"
+	"code.cloudfoundry.org/cli/util/configv3"
+	"code.cloudfoundry.org/cli/util/ui"
+	"errors"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+)
+
+var _ = Describe("create-service Command", func() {
+	var (
+		cmd             *v7.CreateServiceCommand
```

Comment:
The command has a value receiver, but in the tests we are using a pointer. It would be better to be consistent, and I think other commands generally go with a value approach.